domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -152,6 +152,17 @@ constructor(
         return availableCameraLens
     }
 
+    private suspend fun updateMaxFps(currentTargetFrameRate: TargetFrameRate) {
+        coroutineScope {
+            val maxFps = cameraProvider.availableCameraInfos.maxOf { cameraInfo ->
+                cameraInfo.supportedFrameRateRanges.maxOf { e ->
```

Comment:
As mentioned in my other comment, we should populate a list of:
`supportedFramerates: List<Int>` where we add it when `listOf(15, 30, 60).contains(e) && (e.upper == e.lower)`.

Reply:
If we want to be more explicit about these being fixed frame rates, we could call it `supportedFixedFrameRates`.